tunnel/app.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask
+import json
+import sys
+
+app = Flask(__name__)
+
+@app.route("/")
+def api():
+    return (json.dumps(
+            {"left" : {"up" : True, "inkLevel" : 60},
+             "right": {"up" : True, "inkLevel" : 70}}),
+            200,
+            {'Content-Type' : 'application/json'})
```

Comment:
lets do clever stuff like this when we actually build out communication with printers (#29)
```suggestion
    return "Hello from Quasar!"
```